ansible_base/lib/utils/views/urls.py

```diff
@@ -0,0 +1,29 @@
+from typing import Type, Union
+
+from django_extensions.management.commands.show_urls import Command as ShowUrlsCommand
+from rest_framework.views import APIView, View
+
+
+def get_view_functions(urlpatterns) -> set[Union[Type[APIView], Type[View]]]:
```

Comment:
In other services we use Rest Service UrlEnumerator  ti get the active URLs for the service differently from the `ShowUrlsCommand`, this might enable you to not have to pass in urlpatters and only need to run this once for the entire application.

Reply:
The `EndpointEnumerator` appears to work, but it filters out non-APIViews. The result is cleaner though.

Reply:
I would say that this tradeoff is worth it, so I'm applying the changes. Feel free to reopen if you believe otherwise.